.github/workflows/restore-backup-dev-pr.yml

```diff
@@ -0,0 +1,46 @@
+name: DB Restore for dev and feature branches
+
+on:
+  workflow_dispatch:
+    inputs:
+      restoreSourceFile:
+        type: string
+        description: |
+          The sourceFile of the backup in the bucket to restore, e.g.
+          ecamp3-dev/2023-07-17-12-55-33-ecamp3-dev.sql.gz.gpg
```

Comment:
How will we know what to enter here? Is it possible to mistakenly restore prod on dev this way?

Reply:
This is only for dev and pr, and on dev you don't have the access rights and the encryption key to use a prod or staging backup